L1Trigger/L1THGCal/src/backend/HGCalMulticlusteringHistoImpl.cc

```diff
@@ -25,16 +25,19 @@ HGCalMulticlusteringHistoImpl::HGCalMulticlusteringHistoImpl( const edm::Paramet
 	<< "Unknown Multiclustering type '" << multiclusterAlgoType_;
     } 
 
-    edm::LogInfo("HGCalMulticlusterParameters") << "Multicluster dR: " << dr_;  
-    edm::LogInfo("HGCalMulticlusterParameters") << "Multicluster minimum transverse-momentum: " << ptC3dThreshold_;
-    edm::LogInfo("HGCalMulticlusterParameters") << "Multicluster number of R-bins for the histo algorithm: " << nBinsRHisto_<<endl;
-    edm::LogInfo("HGCalMulticlusterParameters") << "Multicluster number of Phi-bins for the histo algorithm: " << nBinsPhiHisto_<<endl;
-    edm::LogInfo("HGCalMulticlusterParameters") << "Multicluster MIPT threshold for histo threshold algorithm: " << histoThreshold_<<endl;
-    edm::LogInfo("HGCalMulticlusterParameters") << "Multicluster type of multiclustering algortihm: " << multiclusterAlgoType_;
+    edm::LogInfo("HGCalMulticlusterParameters") << "Multicluster dR: " << dr_
+    <<"\nMulticluster minimum transverse-momentum: " << ptC3dThreshold_;
```

Comment:
the semicolons `;` have to be removed. This is one single statement spread over mutliple lines.
Also better to use only `\n` character instead of mixing `\n` end `endl`